feat(#765): k8s challenge with securestring (#818)
OWASP/wrongsecrets: src/main/resources/explanations/challenge33_reason.adoc

*Why you should be careful with migrating secrets*

When you update a Kubernetes secret and migrate to another format, the original data is part of the metadata captured in the original configuration.

> Migration of secrets to "something else" can better have a rotation step as well, in order to make sure that the older lesser protected value, is now useless.ch

Reviewer: ```suggestion
Migration of secrets to "something else" should also include a rotation step to ensure the older (less well-protected) value is now useless.
```
